src/ui/components/Icon/styles.scss

```diff
@@ -40,6 +40,16 @@
   width: 112px;
 }
 
+.Icon-triangle-down {
+  background: url("./triangle-down.svg") center no-repeat;
```

Comment:
Oh, maybe. Right now we don't handle SVGs well so this could be a different colour... I'll double-check it.